client/widgets/test-execution-time/time.template.html.slim

```diff
@@ -0,0 +1,30 @@
+.test-execution-time-widget.panel.panel-default
+  .panel-heading
+    help-button
+      p This widget shows the execution time for this specific test in chronological order (from older to newer). By default, only the latest {{ pageSize }} results are shown. If you choose to load older results, they will be added to the beginning of the curve as they are older. Only a maximum of {{ maxResults }} results can be shown.
+      p By default, results are shown for all versions of the project. You may use the version dropdown to see only the test results received for a specific version. If there are versions for which the test was never run, they will not be available to filter.
```

Comment:
"By default, the execution time history is shown across all versions of the project. You may use the version dropdown to display only the history of a specific version. If there are versions for which the test was never run, they will not be available to filter."